amm/src/main/scala/ammonite/interp/ClassLoaders.scala

```diff
@@ -118,7 +118,7 @@ class SpecialClassLoader(parent: ClassLoader, parentSignature: Seq[(Path, Long)]
   }
 
   private def jarSignature(url: URL) = {
-    val path = Path(url.getFile, root)
```

Comment:
What is the purpose of this change? You obviously thought it was interesting enough to mention in the description but you didn't actually say why it was necessary